### ISSUE TITLE
Add more example programs

### ISSUE THREAD

**Issue by PeterTillema** [PeterTillema/ICE#36]
Which ones?

**Comment by LAX18** [comment]
If you are still looking for example programs. I would be more than happy to code a few that showcase certain functions.

**Comment by PeterTillema** [comment]
If you want to, yes please. It should be rather small examples featuring the following topics:
- Appvars
- Sprites
- Displaying text
- Some custom tokens
- Some fancy math expressions
- Lists

More?

**Comment by jcgter777** [comment]
How about matrices when those come out? (or have they?)

**Comment by LAX18** [comment]
@PeterTillema I can certainly do that!

**Comment by PeterTillema** [comment]
>  How about matrices when those come out? (or have they?)

No, since ICE v3.0 isn't out yet, and it takes some time to finish it.

>  @PeterTillema I can certainly do that!

Thanks :)
